conf/rabbitmq.conf

```diff
@@ -0,0 +1,5 @@
+loopback_users.guest = true
```

Comment:
I coun't loggin to rabbit with these credentials, had to use guest/guest, can you please check this?

Reply:
I was looking at it, is not taking the new configuration